### DISCUSSION TITLE
gcp_stackdriver sink api_key?

### DISCUSSION THREAD

**Discussion by phact** [vectordotdev/vector#7505] [Q&A] [answered]
I see some references to `api_key` in the stackdriver docs here https://vector.dev/docs/reference/configuration/sinks/gcp_stackdriver_metrics/#gcp-authentication
but it seems like this functionality is really only implemented for the pubsub sink?

**Comment by jszwedko** [comment, marked as answer]
Ah, you are right @phact . That config option is not supported for the stackdriver metrics sink. I think https://github.com/timberio/vector/issues/7180 will fix this.

**Reply by phact** [reply]
Awesome. Really enjoying using vector btw, thanks for all the hard work!
